src/test/java/net/mcreator/integration/ui/ModElementUITest.java

```diff
@@ -122,7 +122,7 @@ public class ModElementUITest {
 							"dependencies", new ArrayList<String>()).putMetadata("return_type", "ACTIONRESULTTYPE"));
 		}
 
-		for (int i = 1; i <= 2; i++) {
+		for (int i = 1; i <= 1; i++) {

```

Comment:
You should also have changed the value in `GTSampleElements` to make this work everywhere.

Reply:
Both have number 1, don't they?

Reply:
Nerdy didn't touch `GTSampleElements` when reverting this change, the value of one there with value of two here was what caused tests to "not like multiple entities".

Reply:
So it is ok now given only one entity is used in tests now